css/css-flexbox/META.yml

```diff
@@ -133,3 +133,111 @@ links:
   results:
   - test: table-as-item-auto-min-width.html
     status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=210144
+  results:
+  - test: anonymous-flex-item-004.html
+    status: FAIL
+  - test: anonymous-flex-item-005.html
+    status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=210465
+  results:
+  - test: flex-wrap-002.html
+    status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=209871
+  results:
+  - test: flex-wrap-005.html
+    status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=145176
+  results:
+  - test: flexbox_align-items-stretch-3.html
+    status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=210474
+  results:
+  - test: flexbox_first-letter.html
+    status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=136754
```

Comment:
Seems to be a generic 'these tests fail' bug. Was the root failure difficult to determine here?

Reply:
This tests (`css/css-flexbox/flexbox_stf-table-singleline.html` and `css/css-flexbox/flexbox_stf-table-singleline-2.html`) were added to WPT in 2012 by Opera in https://github.com/web-platform-tests/wpt/commit/1a21fff61755062476e4851f8ba8366ab2e22d9e
The failure on Safari its an unexpected blue/red background:
* [Safari failure for css/css-flexbox/flexbox_stf-table-singleline.html](https://wpt.fyi/analyzer?screenshot=sha1%3A034d36646df057ddd5e8ff3ccd330431db7a0c65&screenshot=sha1%3A4693e37e925647c4bc3c064991bae5cbcec2b911)
* [Safari failure for css/css-flexbox/flexbox_stf-table-singleline-2.html](https://wpt.fyi/analyzer?screenshot=sha1%3A1585ef705a01ac8899c12877d8fb214a3d58a039&screenshot=sha1%3A4693e37e925647c4bc3c064991bae5cbcec2b911)

This tests lack a "content" keyword so it's not documented what they exactly test.
So it is not obvious to me what they are exactly testing or why they fail on WebKit.
However I found that bug on WebKit bugzilla where those failures were dectected years ago, so I assigned this failures to this bug. This bug its used on the WebKit test expectations to track the expected failure on WebKit layout tests for the test `css3/flexbox/csswg/flexbox_stf-table-singleline.html` (which its the same than this ones and should be de-duplicated once we import the WPT css-flexbox tests into WebKit)

Reply:
To make the bug less generic, I added a comment on https://bugs.webkit.org/show_bug.cgi?id=136754#c4 regarding this two WPT tests there stating my thoughts about this.